src/main/java/edu/miu/cs/cs544/EAProject/domain/City.java

```diff
@@ -9,7 +10,7 @@
 
 @Entity
 @NoArgsConstructor
-@Getter
```

Comment:
I just wanna allow some properties to change the value in runtime. That's why I didn't use @Data